Kitodo/src/main/java/org/kitodo/production/services/file/FileService.java

```diff
@@ -1183,6 +1183,11 @@ private void addNewURIsToExistingPhysicalDivisions(Map<String, Map<Subfolder, UR
     private void addNewMediaToWorkpiece(List<String> canonicals, Map<String, Map<Subfolder, URI>> mediaToAdd,
             Workpiece workpiece) {
 
+        LogicalDivision actualLogicalRoot = workpiece.getLogicalStructure();
+        while (Objects.isNull(actualLogicalRoot.getType()) && actualLogicalRoot.getChildren().size() == 1) {
```

Comment:
What if there are no childrens yet? (is this possible?) Or when a pre-structured import already brings multiple children?

Reply:
Okay, this isn’t fullly think about.  Case of multiple shildren should be as previous